library-extensions/src/main/java/com/mikepenz/fastadapter_extensions/utilities/SubItemUtil.java

```diff
@@ -305,6 +305,54 @@ else if (!subItemsOnly && getParent(item) == null) {
         return deleted;
     }
 
+    /**
+     * notifies sub items, if they are currently extended
+     *
+     * @param adapter the adapter
+     * @param identifiers set of identifiers that should be notified
+     */
+    public static <T extends IItem & IExpandable> void notifyItemsChanged(final FastAdapter adapter, Set<Long> identifiers) {
+        int i;
+        IItem item;
+        for (i = 0; i < adapter.getItemCount(); i++) {
+            item = adapter.getItem(i);
+            if (item instanceof IExpandable) {
+                notifyItemsChanged(adapter, (T) item, identifiers, true);
+            }
+            else if (identifiers.contains(item.getIdentifier())) {
+                adapter.notifyAdapterItemChanged(i);
+            }
+        }
+    }
+
+    /**
+     * notifies sub items, if they are currently extended
+     *
+     * @param adapter the adapter
+     * @param header the expandable header that should be checked (incl. sub items)
+     * @param identifiers set of identifiers that should be notified
+     * @param checkSubItems true, if sub items of headers items should be checked recursively
+     */
+    public static <T extends IItem & IExpandable> void notifyItemsChanged(final FastAdapter adapter, T header, Set<Long> identifiers, boolean checkSubItems) {
```

Comment:
What do you mean here? The header that should be checked needs to be an `IExpandable`, why don't force this? Then I have to check the header before retrieving it's children...